pkg/storage/gcp/gcp.go

```diff
@@ -45,6 +45,14 @@ func NewWithCredentials(ctx context.Context, cred option.ClientOption) (*Storage
 	}, nil
 }
 
+func newWithBucket(bkt Bucket) *Storage {
```

Comment:
We will have to export this if we want to use a common test suite.

Reply:
That's correct. That could be changed after exporting it into the common test suite though